packages/core/src/components/graphs/bar-stacked.ts

```diff
@@ -169,43 +174,48 @@ export class StackedBar extends Bar {
 
 		const { groupMapsTo } = this.model.getOptions().data;
```

Comment:
Line 173 - 175 should be  nested within the if condition similar to area and area-stacked